webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx

```diff
@@ -287,7 +219,7 @@ function EntityDialog(props) {
     }
 
     return (
-        <Dialog open={!!entityId || (open && entity.isNewEntity)} maxWidth={"xl"} fullWidth>
+        <Dialog open={!!entityId || (open && !entityId)} maxWidth={"xl"} fullWidth>
```

Comment:
```suggestion
        <Dialog open={!!entityId || open} maxWidth={"xl"} fullWidth>
```

Würde das nicht sogar so reichen?